packages/datadog-instrumentations/src/http/server.js

```diff
@@ -31,6 +32,8 @@ function wrapResponseEmit (emit) {
 
     if (eventName === 'finish') {
       finishServerCh.publish({ req: this.req })
+    } else if (eventName === 'close') {
```

Comment:
We need this event because `finish` event is not fired when the request is canceled by the browser. 

For example, multiple _refresh_ in the browser before finish the request